proxy_mobile/magic-script/MagicScript.js

```diff
@@ -1,11 +1,12 @@
 import { AppRegistry } from 'react-native';
-import { ReactNativeMagicScript, PlatformInformation } from 'magic-script-components-react-native';
+import { ReactNativeMagicScript, PlatformInformation, NativePlaneDetector } from 'magic-script-components-react-native';
```

Comment:
How about using prefix `ReactNative`?
`ReactNativeMagicScript`, `ReactNativePlatformInformation`, `ReactNativePlaneDetection`?

Reply:
I'll prepare this change as separate PR.